### DISCUSSION TITLE
*Ajuda* Plataforma para impactados pelo COVID

### DISCUSSION THREAD

**Discussion by RunnableLian** [frontendbr/forum#1742] [Discussão]
Pessoal, estamos buscando ajuda para erguer 1 plataforma open source e sem fins lucrativo, para auxiliar as pessoas impactadas pelo COVID 19 e também pequenos estabelecimentos que estão sofrendo mais.
A ideia é linkar quem está precisando de ajuda com quem pode ajudar através de doações, cestas básicas e afins.
Em um primeiro momento a ideia é direcionar as doações para ONGS(validadas), que serão as responsáveis por receber a doação, e gerir a distribuição das doações para um grupo de familia.
Em um segundo momento(fluxo ainda a ser definido), a ideia é a transferência do ajudante ser para o próprio mercado pequeno que está sendo impactado, e este direcionar a cesta básica a família que está precisando de ajuda.
O projeto está publicado em https://github.com/SOS-Covid.
A stack definida foi nodejs com express no backend, e no frontend react.
Banco de dados mongodb.
Aplicações rodando no Heroku.
Estamos precisando de apoio neste momento, principalmente no front-end, mas toda ajuda é bem vinda.
Fiquem à vontade para clonar o projeto, abrir issues e nos ajudar com sugestões. 
Também estamos aceitando dicas para organização de uma forma melhor no github, bem como de documentação. :)
Valeu pessoal, não deixem de dar uma olhada --> **https://github.com/SOS-Covid **

**Comment by matheus-vieira** [comment]
@LianRolim eu montei esse site para as pessoas solicitarem itens/serviços para estabelecimentos.

[Solicitar](https://matheus-vieira.github.io/solicitar/)
Ele é bem simples e somente possui front-end, não existe nada no back-end.

O que precisa são de dados de estabelecimentos..

Se esse site puder ajudar entre em contato comigo.
